pages/patterns/select-language/language-preferences.md

```diff
@@ -134,7 +134,7 @@ The pattern allows users to select their language preferences for both written a
 - Improving access to public websites and digital services for limited English proficient (LEP) persons. (December 2021) Retrieved on July 25, 2022, from <https://www.lep.gov/sites/lep/files/media/document/2021-12/2021_12_07_Website_Language_Access_Guide_508.pdf>
 - LEP.gov. (n.d.) Retrieved on July 25, 2022, from <https://www.lep.gov/>
 - The limited English proficient population in the United States in 2013. (July 8, 2015) Retrieved on July 25, 2022, from <https://www.migrationpolicy.org/article/limited-english-proficient-population-united-states-2013>
-- OMB memorandum m-17-06, 16. (November 2016) Retrieved on July 25, 2022, from <https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf#page=16>
+- OMB memorandum m-17-06, 16. (November 2016) Retrieved on July 25, 2022, from <https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf>
```

Comment:
Should we keep the page number in the citation? It might be helpful to maintain that reference since we ran into errors linking directly to the page.

Alternatively, we can remove the page number from the reference.

```suggestion
- OMB memorandum m-17-06. (November 2016) Retrieved on July 25, 2022, from <https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf>
```

Reply:
@sarah-sch @finekatie We had a couple of questions about this link. A little background: We have had a long-standing error that prevents us from running our automated HTML proofer check on the "select-a-language" directory on our site, related to the "#page=16" originally added to this link  (See [this comment](https://github.com/uswds/uswds-site/pull/2588#discussion_r1563077334) for more detail). We were wondering - is it ok to remove the page reference from the link and just point to the pdf main page? And if so, should we change the link text? I'm wondering if it might help to add the page number to the link text, maybe something like this:

```suggestion
- OMB memorandum m-17-06,  page 16. (November 2016) Retrieved on July 25, 2022, from <https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf>
```

Curious what you think!

Reply:
Just adding a note that this is the lead text, not the link text itself

![image](https://github.com/uswds/uswds-site/assets/25211650/9f6895b8-05e6-4e6a-b042-7e4082a4fef5)

Reply:
> @sarah-sch @finekatie We had a couple of questions about this link. A little background: We have had a long-standing error that prevents us from running our automated HTML proofer check on the "select-a-language" directory on our site, related to the "#page=16" originally added to this link (See [this comment](https://github.com/uswds/uswds-site/pull/2588#discussion_r1563077334) for more detail). We were wondering - is it ok to remove the page reference from the link and just point to the pdf main page? And if so, should we change the link text? I'm wondering if it might help to add the page number to the link text, maybe something like this:
> 
> Curious what you think!

I think it works fine to take it out of the link and have "page 16" after the OMB title. Just "16" by itself would be a little hard to tell, but Amy's change works for me.